bindings/python/meson.build

```diff
@@ -1,45 +1,37 @@
 get_overridedir = '''
-import os
-import sysconfig
-
-libdir = sysconfig.get_config_var('SCRIPTDIR')
-
-if not libdir:
-  libdir = '/usr/lib'
-
-try:
-  import gi
-  overridedir = gi._overridesdir
-except ImportError:
-  purelibdir = sysconfig.get_path('purelib')
-  overridedir = os.path.join(purelibdir, 'gi', 'overrides')
-
-if overridedir.startswith(libdir): # Should always be True..
-  overridedir = overridedir[len(libdir) + 1:]
+import gi
+overridedir = gi._overridesdir
 
 print(overridedir)
```

Comment:
Perhaps just print it directly?

Reply:
Yeah, probably not a bad idea. Just a leftover from the previous approach.